src/components/Form/From/variants/StandardFrom.tsx

```diff
@@ -0,0 +1,47 @@
+import { FromProps } from '..';
+import { InputField } from '../InputField';
+import { AssetSelector } from '../../../Selector';
+import { AssetSelectorOnChange } from '../../../Selector/AssetSelector/helpers';
+import { FromDescription } from '../Description';
+import { AvailableActions } from '../AvailableActions';
+
+export const StandardFrom = ({
+  setSelectedAsset,
+  className,
+  register,
+  max,
+  setValue,
+  assets,
+  selectedAsset,
+  network,
+  assetSuffix,
+  error,
+  customText,
+  readOnly = false,
```

Comment:
Nit: Can we order them somehow? Maybe alphabetically? I know it doesn't matter much and doesn't change anything about the implementation but having some kind of ordering makes it easier to find something _if_ I was looking for a specific prop. I'd appreciate if we could generally try to order long lists of props as also in other components of this project I saw that they get quite large.